### PR TITLE
Show description of each virtual study

### PR DESCRIPTION
**Pull request by jaouad-benassila**

### CODE REVIEW

File: src/components/UserDashboard/SavedQueries/index.js [hunk 1]
```diff
@@ -191,10 +225,18 @@ export const MySavedQueries = compose(
...
                         <Scroll>
-                          {virtualStudies.map(s => (
-                            <Study>
+                          {virtualStudies.map((s, index) => (
+                            <Study key={s.id}>
                               <Row justifyContent="space-between" width="100%">
-                                <StudyLink to={`/explore?id=${s.id}`}>{s.name}</StudyLink>
+                                <div className={`${studyStyle}`}>
+                                  <StudyLink to={`/explore?id=${s.id}`}>{s.name}</StudyLink>
+                                  {virtualStudyLoad(s.id, index, props)}
```
Comment (jberube) [inline src/components/UserDashboard/SavedQueries/index.js]: That cause a query for every virtual studies of the user at each render.

Please move to the appropriate lifecyle event.

File: src/components/UserDashboard/SavedQueries/index.js [hunk 2]
```diff
@@ -78,6 +99,19 @@ const mapStateToProps = state => {
   };
 };
 
+let descriptions = new Array(20);
```
Comment (jberube) [inline src/components/UserDashboard/SavedQueries/index.js]: Why did you use a module variable here? It won't be set when the query returns, so there won't be a render unless something else triggers it.

Please move to the state of the component, or use redux to store the virtual studies info. We currently only store the information of the loaded study in the cohort-builder, but we could store them all now that there is a need for it. See `src\store\reducers\ui\participantEntityPage.js` for an example. Reducers in the `ui` folder are for specific parts of the UI, if we end-up re-using the information, we will move it to a general-purpose reducer later.

**Comment by jberube** [comment]
I've found an endpoint of Riff that allows to get the whole content of a user, that would be more effective than querying for each saved study: 
https://github.com/kids-first/riff/blob/master/src/main/java/bio/overture/riff/controller/RiffController.java#L50

**Comment by aalex** [comment]
This PR had been connected to https://github.com/kids-first/kf-portal-ui/issues/1722 - but the PR has been closed without being merged.

**Pull request closed without merging**